scripts/compose.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Script based on https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/scripts/compose.js
+ */
+
+const fs = require('fs')
+const inquirer = require('inquirer')
+const dedent = require('dedent')
+
+const genFrontMatter = (answers) => {
+  let d = new Date()
+  const date = [
+    d.getFullYear(),
+    ('0' + (d.getMonth() + 1)).slice(-2),
+    ('0' + d.getDate()).slice(-2),
+  ].join('-')
+  const tagArray = answers.tags.split(',')
+  tagArray.forEach((tag, index) => (tagArray[index] = tag.trim()))
+  const tags = "'" + tagArray.join("','") + "'"
+
+  let frontMatter = dedent`---
+  title: ${answers.title ? answers.title : 'Untitled'}
+  date: '${date}'
+  type: ${answers.type}
+  tags: [${answers.tags ? tags : ''}]
+  cover: /img/posts/may-2021-at-asyncapi/cover.webp
+  draft: ${answers.draft === 'yes' ? true : false}
```

Comment:
you can remove it, we do not support it here, like nothing will happen if you have it in front matter or not

Reply:
what about the `type`, should I remove it too?

Reply:
no no, type is used, is important. It is consumed in UI as one of filters, and also shows as a kind of article type name

![Screenshot 2022-01-27 at 21 11 01](https://user-images.githubusercontent.com/6995927/151436099-13c33636-4750-43b5-a1ce-bffcd42c3588.png)

Reply:
so the one I should remove is just `draft`?

Reply:
@derberg I removed the `draft` option 👍